tests/chia_log/handlers/test_wallet_added_coin_handler.py

```diff
@@ -34,6 +34,15 @@ def testFloatPrecision(self):
         self.assertEqual(events[0].service, EventService.WALLET, "Unexpected service")
         self.assertEqual(events[0].message, "Cha-ching! Just received 0.000000000001 XCH ☘️")
 
+    def testTransactionAmountFilter(self):
+        filter_handler = WalletAddedCoinHandler(config={
+            'transaction_amount': "0.000000000001"
+        })
+        with open(self.example_logs_path / "small_values.txt", encoding="UTF-8") as f:
+            logs = f.readlines()
+        events = filter_handler.handle("".join(logs))
+        self.assertEqual(0, len(events))
```

Comment:
suggestion: let's add at least 1 value above the threshold in the example file, so we can hit both below/above threshold in the same test?